includes/class-alg-wc-checkout-fees.php

```diff
@@ -549,7 +549,7 @@ public function calculate_the_fee( $args, $final_fee_to_add, $total_in_cart, $fe
 				$max_fee = 0;
 			}
 			// Max fee.
-			if ( 0 != $max_fee && $new_fee > $max_fee ) { //phpcs:ignore
+			if ( 0 != $max_fee && $new_fee < $max_fee ) { //phpcs:ignore
```

Comment:
@mokshasharmila13 This is incorrect. This will not work when the fee value & max fee value is a positive value. You need to check for both positive & negative values